src/components/select/select.scss

```diff
@@ -228,3 +228,41 @@ md-optgroup {
     border: 1px solid #fff;
   }
 }
+
+md-select-menu[multiple] {
+    md-option {
+        height: $select-option-height;
+        padding: 0 $select-option-padding 0 ($select-option-padding / 2);
+        padding-left: ($select-option-padding * 2) + ($select-option-padding / 2);
+
+        &:before {
```

Comment:
@DerekLouie Why are we not using the `md-checkbox` component? I know it will be a bit more complex, to integrate the `md-checkbox` component, but I think this would be better way, instead of mirroring the checkbox. Thoughts?